x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Mod.java

```diff
@@ -12,7 +12,7 @@
 /**
  * <a href="https://en.wikipedia.org/wiki/Modulo_operation">Modulo</a>
  * function ({@code a % b}).
- * 
+ *
  * Note this operator is also registered as a function (needed for ODBC/SQL) purposes.
  */
 public class Mod extends ArithmeticOperation {
```

Comment:
I think the `inverse` should throw an exception in this class.

Reply:
See above [comment](https://github.com/elastic/elasticsearch/pull/66022#discussion_r553452382).